appgate/appgate.py

```diff
@@ -219,8 +223,11 @@ async def get_current_appgate_state(ctx: Context) -> AppgateState:
     if ctx.no_verify:
         log.warning('[appgate-operator/%s] Ignoring SSL certificates!',
                     ctx.namespace)
+    if ctx.device_id is None:
+        raise AppgateException('No device id specified')
```

Comment:
Can we even reach this code?

Reply:
In theory no, so this is mainly to satisfy the type checker.